### ISSUE TITLE
Following update to version 2.3.0 Youtube videos only play sound and no video

### ISSUE THREAD

**Issue by digitalred** [piroor/treestyletab#1640]
## Short description
Following an update to version 2.3.0 today, Youtube videos no longer play video, but do play sound.  Disabling the "TreeStyleTab" allows videos to once again play. 
## Steps to reproduce
1.  Install TST in Firefox 57
2.  Enable
3.  Go to Youtube 
4.  Click on any Video (no video but sound is heard)
5.  Disable TST to allow Video to play

 

## Expected result
Expect video to play and sound to play

## Actual result

No Video but sound is audible

## Environment

 * Platform (OS): MacBook Pro Late 2014 OS 10.11.6
 * Version of Firefox: 57.0
 * Version (or revision) of Tree Style Tab: 2.3.0

**Comment by romeoro** [comment]
Can confirm on MacBook Pro Mid 2012 with OS 10.13.1 with FF 57.0.1.
Videos do not play at all visually or only for a very short moment and then freeze.
Audio continous normally.
Only happens with TST installed.

**Comment by piroor** [comment]
This doesn't happen on my Windows 10. Hmm...

**Comment by piroor** [comment]
Doesn't happen also on macOS.

 * Firefox 57.0.1 (20171128222554)
   * Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:57.0) Gecko/20100101 Firefox/57.0
   * Darwin 15.6.0
   * e10s enabled
 * TST 2.3.0

**Comment by piroor** [comment]
Does this depend on macOS version?

**Comment by digitalred** [comment]
I believe I have identified the possible issue.  After further testing I discovered that the sidebar impacted the presence of the Video window.  If I closed the Sidebar the video reappeared and when I opened it, the video window would disapear again.  I generally run my browser window at around 85% of of full screen and when I enlarged the window to full screen the video reappeared with the sidebar in place. When I returned the windows to the 85% size the video disappeared.  I did this numerous times.   I then manually resized the windows several times and all of a sudden it began to work regardless of what size the window was, infact it began to dynamcally re-adjust the video and the rest of the content on the screen to accomodate the size change.  Looks like it is resolved, although I am unclear why it happened in the first place.
